### PR TITLE
Adding custom inlist delimiter logic

### PR DESCRIPTION
**Pull request by pjohn1024**

### CODE REVIEW

File: model/entity/Collection.cfc [hunk 1]
```diff
@@ -3699,7 +3709,7 @@ component displayname="Collection" entityname="SlatwallCollection" table="SwColl
 		if(len(arguments.comparisonOperator)){
 			key &= ":#arguments.comparisonOperator#";
 		}
-		var delimiter = ',';
+		var delimiter = this.getInlistDelimiter();
```
Comment (migueltarga) [inline model/entity/Collection.cfc]: Can you use other delimiters on IN filter in HQL?

File: model/entity/Collection.cfc [hunk 2]
```diff
@@ -2773,7 +2783,7 @@ component displayname="Collection" entityname="SlatwallCollection" table="SwColl
 			predicate = filter.value;
 		}else if(arguments.filter.comparisonOperator eq 'in' || arguments.filter.comparisonOperator eq 'not in'){
 			if(len(filter.value)){
-				predicate = "(" & ListQualify(filter.value,"'") & ")";
+				predicate = "(" & ListQualify(filter.value,variables.inlistDelimiter) & ")";
```
Comment (migueltarga) [inline model/entity/Collection.cfc]: this doesn't look right, the delimiter should be the 3rd parameter

File: model/entity/Collection.cfc (hunk 1 above)
Comment (pjohn1024) [reply model/entity/Collection.cfc]: @migueltarga I think we are not changing the delimiter on the IN Filter in HQL, The new delimiter is Just used for parsing the in filter list in the predicate

File: model/entity/Collection.cfc (hunk 2 above)
Comment (pjohn1024) [reply model/entity/Collection.cfc]: Fixed

**Pull request merged**